src/TorchSharp/Tensor/Tensor.Math.cs

```diff
@@ -2754,7 +2754,14 @@ public static Tensor einsum(string equation, params Tensor[] tensors)
         /// </summary>
         /// <param name="input">The first input tensor</param>
         /// <param name="other">The second input tensor</param>
-        public static Tensor maximum(Tensor input, Tensor other) => input.maximum(other);
+        public static Tensor max(Tensor input, Tensor other) => maximum(input, other);
```

Comment:
I don't think 'max' and 'maximum' are aliases of each other.

https://pytorch.org/docs/stable/generated/torch.max.html?highlight=max#torch.max
https://pytorch.org/docs/stable/generated/torch.maximum.html?highlight=maximum#torch.maximum

Reply:
there is an overload at the bottom of the 'max‘ page

torch.max(input, other, *, out=None) → [Tensor](https://pytorch.org/docs/stable/tensors.html#torch.Tensor)
See [torch.maximum()](https://pytorch.org/docs/stable/generated/torch.maximum.html#torch.maximum).